### ISSUE TITLE
Using Preferences > Sound System to change the input device twice breaks accessibility support

### ISSUE THREAD

**Issue by dglee42** [BearWare/TeamTalk5#2636]
### Description

In this TeamTalk beta version (5.18.0.5161), if you use Preferences > Sound System to change the input device, then do it again, accessibility support stops working. Both JAWS and NVDA can no longer read controls accurately until TeamTalk is restarted. This likely also applies if changing the output device. This problem did not occur in the previous beta - 5.18.0.5160 I think. It also does not occur if you use the menus instead of Preferences > Sound System to change the active sound device.

### Application

- [x] qtTeamTalk
- [ ] TeamTalkAndroid
- [ ] iTeamTalk
- [ ] TeamTalkClassic
- [ ] TeamTalkServer

### Platform

- [x] Windows
- [ ] macOS
- [ ] Android
- [ ] iOS
- [ ] Linux

### Expected behavior

Using Preferences > Sound System any number of times should not alter accessibility support for the application.

### Actual behavior

After the second device change, most or all controls use the title bar as the control name, UIA seems confused as to where focus is, and screen readers are unable to report controls correctly.

### Steps to reproduce problem

Open TeamTalk and enter a server (we did not prove whether an active channel is necessary, but we were always in one).

Use F4 to open Preferences, then move to the Sound System tab and change the active input device. Press Enter when done, to effect the change and return to the main screen. Repeat this step a second time.

Try to use Tab to move among controls in the main screen with JAWS or NVDA. If possible, use a tool to examine the UIA accessibility tree for the application, and determine what UIA says has focus at various times.

**Comment by CoBC** [comment]
Hmmmm, I cannot reproduce this :(

**Comment by dglee42** [comment]
Very interesting. I and one other person produced it several times last night, in the specific TeamTalk Qt Windows version I gave. He found it, then I reproduced it a few times from his steps.
